### DISCUSSION TITLE
Suggestion: Minimize to Taskbar

### DISCUSSION THREAD

**Discussion by AnnikenYT** [MarcoPNS/Spotify-Stream-Widget#75] [Ideas]
I think this is a very great tool for streamers, but that a feature is missing to minimize the program to the taskbar.

**Comment by MarcoPNS** [comment]
Do you mean the entire form that is visible on stream?
Unfortunately this is technically not possible with the windows version. 

With the web version this is not a problem anymore but this is still WIP

**Comment by Shakesbeard** [comment]
Well, I would like to have a minimize as well. I am aware that capturing might be an issue but I am not using the player window at all. I only use the exportede details to create a custom layout on stream.

**Comment by MarcoPNS** [comment]
@Shakesbeard ok that's a different case. I could add an option to deactivate the viewer entirely for the users that just use the raw data. Let me think about it.

**Comment by Shakesbeard** [comment]
> @Shakesbeard ok that's a different case. I could add an option to deactivate the viewer entirely for the users that just use the raw data. Let me think about it.

That would be nice. Thanks a bunch
